examples/with-vitest/turbo.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "ui": "tui",
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": ["$TURBO_DEFAULT$", ".env*"],
+      "outputs": [".next/**", "!.next/cache/**", "dist/**"]
+    },
+    "lint": {
+      "dependsOn": ["^lint"]
+    },
+    "check-types": {
+      "dependsOn": ["^check-types"]
+    },
+    "dev": {
+      "cache": false,
+      "persistent": true
+    },
+    "test": {
+      "dependsOn": ["^test", "@repo/vitest-config#build"],
```

Comment:
We should probably switch to `topo` so if users add tests to `@repo/ui` then `web#test` and `docs#test` don't block on those finishing:
![graphviz(2)](https://github.com/user-attachments/assets/cc97dbfb-92dc-433b-bd23-f39a19d8f264)